components/supported/SupportedExchanges.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="py-12">
+    <heading>Supported Exchanges</heading>
+    <div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-4 mt-8">
```

Comment:
These two components seem to follow the same pattern. Maybe it makes sense for the repeating logic could become a new component